angular.json

```diff
@@ -26,7 +26,7 @@
               "src/ndb-theme.scss",
               "src/styles.css"
             ],
-            "scripts": []
+            "scripts": ["node_modules/marked/lib/marked.js"]
```

Comment:
why do we need this? (i.e. what does it actually do?)

Reply:
As the library is using marked parser you will need to add node_modules/marked/lib/marked.js to your application.

If you are using Angular CLI you can follow the angular.json example below...

"scripts": [
+ "node_modules/marked/lib/marked.js"
]